src/api/rest/providers/github/schemas/orgs.js

```diff
@@ -0,0 +1,152 @@
+import { schema } from 'normalizr';
```

Comment:
This schema is the one extensively & accurately (no copy/paste) documented.

Private flags setting should be mututalized between all schemas with a helper function: 
const processed = initSchema(entity.id); // { _isComplete: false, _isAuth: false, _entityUrl: false, _fetchedAt: `now`, id: `the passed id`